packages/otp2-tile-overlay/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@opentripplanner/otp2-tile-overlay",
+  "version": "1.0.0",
+  "description": "Render data from OTP2's vector tile server",
+  "main": "lib/index.js",
+  "module": "esm/index.js",
+  "types": "lib/index.d.ts",
+  "repository": "https://github.com/opentripplanner/otp-ui.git",
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "tsc": "tsc"
+  },
+  "peerDependencies": {
+    "react": "^16.14.0",
+    "react-map-gl": "^7.0.15"
+  },
+  "dependencies": {
+    "@opentripplanner/stops-overlay": "^5.0.3-alpha.1",
+    "@opentripplanner/vehicle-rental-overlay": "^2.1.0-alpha.1"
+  },
```

Comment:
Hmm I am not sure I like the dependency to these overlays. Intuitively, it looks like it should be the other way around instead?

Reply:
I've added a comment that the overlays should be deprecated once OTP1 support is removed, and the overlays merged into a new one that lives inside the OTP2 tile layer.